generate_markdown_from_doxygen_xml.py

```diff
@@ -0,0 +1,1031 @@
+        self.id = aTag.attrib['id']
+        self.prot= aTag.attrib['prot']
+        self.static = aTag.attrib['static']
+        self.kind = aTag.attrib['kind']
+        self.mutable = aTag.attrib['mutable']
+
+        self.type = aTag.find('type')
+        self.definition = aTag.find('definition')
+        self.argsstring = aTag.find('argsstring') 
+        self.initializer = aTag.find('initializer')
+        self.briefdescription = aTag.find('briefdescription')  # Needs for processing
+        self.detaileddescription = aTag.find('detaileddescription') # Needs for processing
+        self.inbodydescription = aTag.find('inbodydescription') # Needs for processing
+        
+    def markdown_overview(self):
+        #print('debug:start:cppAttribute:markdown_overview')
+        static_string=''
+        if self.static=='yes':
+            static_string='static'
+        output_string=''
+        #print('debug: static_string: %s' % static_string)
+        #print('debug: self.type: %s' % self.type)
+        #print('debug: self.name: %s' % self.name)
+        #print('debug: self.initializer: %s' % self.initializer)
+        #print('debug: self.briefdescription: %s' % markdown_any_tag(self.briefdescription).strip() )
+        
+        output_string+='\n\n%s %s [%s](#%s)%s - %s\n\n' % (static_string,markdown_any_tag(self.type).strip(), self.name,self.id,markdown_any_tag(self.initializer).strip(),markdown_any_tag(self.briefdescription).strip() )
+        return output_string
+        
+        
+    def markdown(self):
+        """
+        static_string=''
+        if self.static=='yes':
+            static_string='static '
+        """
+        output_string=''
+        output_string+='\n\n### %s {#%s}\n' % (self.name,self.id)
+        output_string+='\n```cpp\n%s%s\n```\n' % (markdown_any_tag(self.definition).strip(),markdown_any_tag(self.initializer).strip())
+
+        output_string+='\n\n%s' % markdown_any_tag(self.briefdescription).strip()
+        output_string+='\n\n%s' % markdown_any_tag(self.detaileddescription).strip()
+
+        if args.debug:
+	    output_string+='\n\n<!-- [%s %s](#%s) -->' % (self.type, self.name,self.id)
+	    output_string+='\n<!-- kind: %s -->' % markdown_any_tag(self.kind).strip()
+	    output_string+='\n<!-- prot: %s -->' % markdown_any_tag(self.prot).strip()
+	    output_string+='\n<!-- static: %s -->' % markdown_any_tag(self.static).strip()
+	    output_string+='\n<!-- mutable: %s -->' % markdown_any_tag(self.mutable).strip()
+	    output_string+='\n<!-- definition: %s -->' % markdown_any_tag(self.definition).strip()
+	    output_string+='\n<!-- detaileddescription: %s -->' % markdown_any_tag(self.detaileddescription).strip()
+	    output_string+='\n<!-- briefdescription: %s -->\n' % markdown_any_tag(self.briefdescription).strip()
+	    output_string+='\n<!-- argsstring: %s -->\n' % markdown_any_tag(self.argsstring).strip()
+
+        return output_string
+                
+        ## xxxxDoFieldToo
```

Comment:
And I don't understand this either? What about `# TODO: blabla.`

Reply:
Deleted. Yes, #TODO would be better :-)